pennylane_lightning/lightning_qubit.py

```diff
@@ -32,12 +33,24 @@
 from ._version import __version__
 
 try:
-    from .lightning_qubit_ops import (
-        apply,
-        StateVectorC64,
-        StateVectorC128,
-        AdjointJacobianC128,
-    )
+    if platform.system() == "Windows" and sys.version_info[:2] >= (3, 8):
+        # Add the current directory to DLL path.
+        # See https://docs.python.org/3/whatsnew/3.8.html#bpo-36085-whatsnew
+        os.add_dll_directory(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
Maybe we can add a Pylint ignore comment here to ignore the warnings. Or, should we add tests as recommended?